lib/vanagon/platform/windows.rb

```diff
@@ -107,8 +109,9 @@ def generate_msi_package(project)
         dir_ref = "INSTALLDIR"
         # Actual array of commands to be written to the Makefile
         ["mkdir -p output/#{target_dir}",
-        "mkdir -p $(tempdir)/#{project.name}/staging",
-        "gunzip -c #{project.name}-#{project.version}.tar.gz | '#{@tar}' -C '$(tempdir)/#{project.name}/staging' --strip-components 1 -xf -",
+        "mkdir -p $(tempdir)/{staging,wix}",
+        "cp -v -r wix/* $(tempdir)/wix",
```

Comment:
Ok - my preference would be to leave the -v in as it doesn't add a lot extra, but since we can easily check the result of the copy command I will remove it.

Did think about using -vr, but since the heat/light commands have a lot -xx -yy type directives (which are odd and not gnu-ish), I though separate options here were better.